conf/distro/nilrt.conf

```diff
@@ -14,7 +14,12 @@ DISTRO_FEATURES:append:x64 = "\
 
 INHERIT += " create-spdx "
 
-VIRTUAL-RUNTIME_xserver_common = "xserver-common"
+# Mesa requires LLVM AMDGPU and SPIRV targets (CMake list, use semicolons).
+LLVM_TARGETS_TO_BUILD:append = ";AMDGPU;SPIRV"
```

Comment:
[nit] Sorry didn't notice this in last round but shouldn't this be x64 specific (like [this](https://github.com/ni/meta-nilrt/blob/nilrt/master/scarthgap/recipes-core/images/nilrt-runmode-rootfs.bb#L13)) as it's graphics related (we don't support GUI on ARM targets)?

Doesn't need to be fixed in this PR - could be a follow up PR/WI.